sdk/storage/src/clients/table_service_client.rs

```diff
@@ -0,0 +1,53 @@
+use crate::clients::{ServiceType, StorageAccountClient};
+use crate::table::requests::*;
+use azure_core::errors::AzureError;
+use http::method::Method;
+use http::request::{Builder, Request};
+use std::sync::Arc;
+
+pub trait AsTableServiceClient {
```

Comment:
Why?

Reply:
Same as above, to implement a function for `Arc<Box<..>>`.